__tests__/app.test.js

```diff
@@ -228,3 +227,87 @@ describe(" Adds new Comment to: api/articles/:article_id/comments", () => {
     })
   })
 });
+
+describe('PATCH articles', ()=>{
+  test('PATCH: 200, increases vote by article_id', ()=>{
+    const newVote = {inc_votes: 5}
+    return request(app)
+    .patch('/api/articles/10')
+    .send(newVote)
+    .expect(200)
+    .then((response)=>{
+      const {article} = response.body
+      expect(article).toMatchObject({
+        article_id : 10,
+        title: 'Seven inspirational thought leaders from Manchester UK',
+        topic: 'mitch',
+        author: 'rogersop',
+        body: "Who are we kidding, there is only one, and it's Mitch!",
+        created_at : expect.any(String),
+        votes: 5,
+        article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+      })
+    })
+  })
+  test('PATCH: 200 decreases the vote by article ID', ()=>{
+    const newVote = {inc_votes: -4}
+    return request(app)
+    .patch('/api/articles/1')
+    .send(newVote)
+    .expect(200)
+    .then((response)=>{
+      const {article} = response.body
+      expect(article).toMatchObject({
+        article_id : 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: expect.any(String),
+        votes: 96,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+      })
+    })
+  })
+  test('ERROR: 404 responds with an error when trying to decrease the vote to non-existing article (decrease)', ()=>{
+    const newVote = {inc_votes: -4}
+    return request(app)
+    .patch('/api/articles/45')
+    .send(newVote)
+    .expect(404)
+    .then(({body})=>{
+      expect(body.msg).toBe('Not Found')
+    })
+  })
+  test('ERROR: 400 responds with error if patching to an invalid path', ()=>{
+    const newVote = {inc_vote: 45}
+    return request(app)
+    .patch('/api/articles/wow')
+    .send(newVote)
+    .expect(400)
+    .then(({body})=>{
+      expect(body.msg).toBe('Bad Request')
+    })
+  })
+  test('ERROR: 400, responds with an error when vote is not a number', ()=>{
+    const newVote = {inc_vote: 'six'}
+    return request(app)
+    .patch('/api/articles/wow')
```

Comment:
Make sure the path is valid so you are definitely testing what you have set out to test